pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/UserRegistrationPayload.java

```diff
@@ -59,11 +59,20 @@ public class UserRegistrationPayload {
     @SerializedName("timeZone")
     private String timeZone;
 
+    //only used for Osteo1 user creation
+    private boolean skipTriggerEvents = false;
```

Comment:
Makes sense to me to put this here, since this route is locked down IP-wise so that only Auth0 can call it (and in any event, the route isn't visible on prod).